concent_api/concent_api/system_check.py

```diff
@@ -256,6 +256,30 @@ def create_error_39_storage_server_internal_address_is_not_set():
     )
 
 
+def create_error_40_verifier_download_chunk_size_is_not_defined():
+    return Error(
+        'VERIFIER_DOWNLOAD_CHUNK_SIZE setting is not defined',
+        hint='Set VERIFIER_DOWNLOAD_CHUNK_SIZE in your local_settings.py.',
+        id='concent.E040',
+    )
+
+
+def create_error_41_verifier_download_chunk_size_has_wrong_type(verifier_download_chunk_size_type):
+    return Error(
+        'VERIFIER_DOWNLOAD_CHUNK_SIZE has wrong type',
+        hint=f"VERIFIER_DOWNLOAD_CHUNK_SIZE must be integer instead of {verifier_download_chunk_size_type}.",
+        id='concent.E041',
+    )
+
+
+def create_error_42_verifier_download_chunk_size_has_wrong_value(verifier_download_chunk_size_value):
+    return Error(
+        'VERIFIER_DOWNLOAD_CHUNK_SIZE setting has wrong value',
+        hint=f"VERIFIER_DOWNLOAD_CHUNK_SIZE must be greater or equal 1. Currently it has {verifier_download_chunk_size_value}.",
```

Comment:
"be greater or equal **than** 1" or just "positive"

Reply:
Done